log/filter.go

```diff
@@ -64,7 +64,19 @@ func (f *Filter) Log(level Level, keyvals ...interface{}) error {
 	if level < f.level {
 		return nil
 	}
-	if f.filter != nil && f.filter(level, keyvals...) {
+	var fkv []interface{}
+	// is use logger and with prefix,
+	// carry them to filer keyvals also.
```

Comment:
LGTM only with some typos, it may occur unavoidable memory copies and I can't see any better solution either :)

```suggestion
	// fkv is used to provide a logger slice to contains both prefix and keyvals for filter
```

Reply:
done